test/integration/entry-integration.js

```diff
@@ -301,7 +301,7 @@ describe('Entry Api', () => {
     })
```

Comment:
A test in this file is failing. `Gets entries by creation order and id order`. I don't know if it's related to your changes though. Rerun the tests maybe?

Reply:
yeah apperently I had to delete the entries and Content types that were created and not deleted in my failing tests so this one works correvtly.